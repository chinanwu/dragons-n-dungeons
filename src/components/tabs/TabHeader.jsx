import React from 'react';
import PropTypes from 'prop-types';

// TODO: still working on the name
export const TabHeader = ({ children }) => {
  return children;
};

TabHeader.propTypes = {
  children: PropTypes.any,
};
export default TabHeader;
