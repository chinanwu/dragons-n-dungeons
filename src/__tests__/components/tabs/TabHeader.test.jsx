import { shallow } from 'enzyme';
import React from 'react';

import { TabHeader } from '../../../components/tabs/TabHeader.jsx';

jest.unmock('../../../components/tabs/TabHeader.jsx');

describe('TabHeader component', () => {
  describe('rendering', () => {
    it('renders TabHeader', () => {
      const wrapper = shallow(<TabHeader />);
      expect(wrapper.getElement()).toMatchSnapshot();
    });
  });
});
