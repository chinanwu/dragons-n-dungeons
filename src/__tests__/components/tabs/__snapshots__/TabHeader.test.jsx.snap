// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabHeader component rendering renders TabHeader 1`] = `null`;
